test/e2e/thanos_querier_test.go

```diff
@@ -91,3 +94,142 @@ func TestThanosQueryCanQueryWatchdogAlert(t *testing.T) {
 		},
 	)
 }
+
+// TestThanosQuerierTenancyAPI ensures that the ThanosQuerier API exposed on the
+// tenancy port enforces the namespace value.
+func TestThanosQuerierTenancyAPI(t *testing.T) {
```

Comment:
yet to verify test

Reply:
I will move the test to https://github.com/openshift/cluster-monitoring-operator/blob/master/test/e2e/user_workload_monitoring_test.go since other tests are over there